### DISCUSSION TITLE
UX suggestion: Having check boxes instead of dropdown for Differences and Slide

### DISCUSSION THREAD

**Discussion by jacobovillegas** [cssbattle/forum#136] [General]
When i am checking differences, i usually go from normal to differences a lot. specially when you have a 99.9%.

It would be nice to have checkboxes instead of dropdown for these two links. I think is not going to hurt space, and maybe a tooltip explaining what each box does.

I'm UX/UI. Maybe i am right or totally wrong. In that case forget this :)

Something like the image attached

<img width="714" alt="Screenshot 2021-10-15 at 16 36 03" src="https://user-images.githubusercontent.com/435754/137505079-7a1f3f51-a2c8-4551-aa83-52c2b42c1993.png">

**Comment by chinchang** [comment]
valid point. We'll do something about this. May be keyboard shortcuts to toggle these?

**Reply by jacobovillegas** [reply]
Yes, shortcut! thank you.

**Reply by gunnarbittersmann** [reply]
Both, please: checkboxes without dropdown and keybord shortcuts. Perhaps also the switch vertical/horizontal sliding in the UI.

**Comment by chinchang** [comment]
The checkboxes have now been brought out of the dropdown now. We'll add keyboard shortcuts soon too! Thanks for your feedback @jacobovillegas 🙌🏼

**Reply by jacobovillegas** [reply]
You guys rock

**Comment by Knusk** [comment]
So, if I try 'difference' on my solution, and it is all black, should that imply 100%? I have a situation with what looks 100% to me, but only gets ~97% ... Actually, now that I look again, I see a faint shadow in the black; meaning the color is wrong. The color is correct, though, so this must be a rendering issue. Firefox on an older MacBook ... Anything I can do? It is Target #88 that causes this ...

**Reply by chinchang** [reply]
You should play on Chromium based browser only.

**Reply by Knusk** [reply]
Yes, but for some (other) reason, Chrome is not playing well on mac these days. 'Privacy error' on a lot of sites, including cssbattle ... I will try with other machine, other browser tomorrow ... Thanks! :)
